parlai/chat_service/services/terminal_chat/changes_to_core.txt

```diff
@@ -0,0 +1,5 @@
+Core Agents.py:
```

Comment:
what is this file?

Reply:
This is a file to keep of track of things we should change in the `chat_services/core` folder, the file will be deleted before merging. So for example, in `chat_services/core/agents.py`, we there should be a `self.message_partners` class variable but currently there isn't.

Reply:
ok, please indicate when this is ready for full review